00-Raster-Structure.Rmd

```diff
@@ -123,7 +133,7 @@ library(rgdal)
 DSM_HARV <- raster("NEON_RemoteSensing/HARV/DSM/HARV_dsmCrop.tif")
 
 # code output here - DEM rendered on the screen
-plot(DSM_HARV, main="NEON Elevation Map\nHarvard Forest")
+plot(DSM_HARV, main="Continuous Elevation Map\nHarvard Forest")
```

Comment:
Changed it to  "Continuous" as it is compared to the next map with is titled "Classified Elevation Map\nHarvard Forest".  Changing both to ...\n NEON Harvard Forest Field Site